electrum/lnpeer.py

```diff
@@ -2734,14 +2734,19 @@ async def wrapped_callback():
                     fut = asyncio.ensure_future(wrapped_callback())
                 # return payment_key so this branch will not be executed again
                 return None, payment_key, None
+            elif preimage:
+                return preimage, None, None
+            else:
+                # we are waiting for mpp consolidation or preimage
+                return None, None, None
         else:
```

Comment:
I see that now if the first part of the `if` is entered, we are guaranteed to return.
Would it be clearer to remove the `else:` and indent everything below one to the left?
Or keep the `else:` and indent everything to the right, to be inside?

Reply:
I think it is clearer to keep the outer `else:`. Note that the inner else branch that executes `return None, None, None` was added for clarity. simply writing `return preimage, None, None` would have been sufficient.

Reply:
ah, yes, I see what you mean, some code can be right-indented.

Reply:
done